src/Blocks/TopicLink.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * TopicLink block class
+ *
+ * @package P4\MasterTheme
+ * @since 0.1
+ */
+
+ namespace P4\MasterTheme\Blocks;
+
+/**
+ * Class TopicLink
+ *
+ * @package P4\MasterTheme\Blocks
+ */
+class TopicLink extends BaseBlock
+{
+    /**
+     * Block name.
+     *
+     * @const string BLOCK_NAME.
+     */
+    public const BLOCK_NAME = 'topic-link';
+
+    /**
+     * Counter constructor.
+     */
+    public function __construct()
+    {
+        register_block_type(
+            self::get_full_block_name(),
+            [
+                'editor_script' => 'planet4-blocks',
+                'render_callback' => [ self::class, 'hydrate_frontend' ],
```

Comment:
If you want to use hydration, you need to implement it in the frontend too with a script using our [hydrateBlock function](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/functions/hydrateBlock.js) (you can look at examples of blocks that use it already, such as the Gallery)
At the moment this will be the same as using `render_frontend`, if you look at the [function's code](https://github.com/greenpeace/planet4-master-theme/blob/main/src/Blocks/BaseBlock.php#L272)

Reply:
Thanks! What do you suggest me to use?

Reply:
Fixed!